src/Date/DatePickerModalContentHeader.tsx

```diff
@@ -62,19 +63,18 @@ export default function DatePickerModalContentHeader(
     editIcon,
     calendarIcon,
     inputDate,
+    allowEditing,
   } = props
   const theme = useTheme()
   const label = getLabel(props.locale, props.mode, props.label)
 
   const color = useHeaderTextColor()
-  const supportingTextColor = theme.isV3 ? theme.colors.onSurfaceVariant : color
-  let allowEditing = mode !== 'multiple'
 
-  if (inputDate !== undefined) {
-    allowEditing = inputDate
-  }
+  const isAllowEditing = allowEditing && mode !== 'multiple' && inputDate !== undefined
```

Comment:
I think this change brought some undesired side effects, as
```
if (inputDate !== undefined) {
    allowEditing = inputDate
}
```
is not equivalent to making `const isAllowEditing = ... && inputDate !== undefined`. 

The first is overriding the value of `isAllowEditing` with `inputDate` if this is defined.
The second is having an impact on the `isAllowEditing` value depending on `inputDate` being defined or not.

At the same time, the existing `inputDate` property is and was never set from anywhere outside where this component is used, so its value is always `undefined`.

All of this make the edit button to always be hidden regardless the value passed for the `allowEditing` prop.

Reply:
Hi @jcloquell, thanks for catching this! Would you mind creating a quick PR to address this?

Reply:
@jcloquell i think can change to this:
`const isAllowEditing = allowEditing && mode !== 'multiple'`

And remove this:
`if (inputDate !== undefined) {
    allowEditing = inputDate
}`

What do you think?

Reply:
Sure! Here is the PR - https://github.com/web-ridge/react-native-paper-dates/pull/264 :)